.github/workflows/test-coverage.yaml

```diff
@@ -0,0 +1,58 @@
+on:
+  push:
+    branches:
+      - master
```

Comment:
In both these places, `master` should be `main`, since this is now used by default for new repositories. This is why the coverage test was not triggered. If you look at https://github.com/r-lib/actions/blob/master/examples/test-coverage.yaml they use both `master` and `main` just to make it work for both conventions.